cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -185,6 +185,79 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'align', 'justify');
 	});
 
+	it('Apply Line spacing: 1', function() {
+		cy.get('#tb_editbar_item_linespacing')
+			.click();
+
+		helper.reselectAllText();
+		
+		cy.get('.menu-text').eq(0)
+			.click();
```

Comment:
As I said earlier, it would be better to use cy.cotnains() here, because it will make the test code more readable and maintainable. Is there any issue with using cy.contains() here, that I'm not aware of?